sdcm/sct_runner.py

```diff
@@ -104,10 +107,10 @@ def install_prereqs(self, public_ip):
             # Jenkins pipelines run /bin/sh for some reason
             unlink /bin/sh
             ln -s /bin/bash /bin/sh
-
+            echo "cd ~/home/ubuntu" >> /home/ubuntu/.bashrc
```

Comment:
Why this is needed? And how is it related to the addition of "kubectl"?

Reply:
Not needed - removed now.